V1/Controllers/WalletController.cs

```diff
@@ -46,5 +46,13 @@ public async Task<IActionResult> GetWalletStatus(string walletId, [FromBody] Wal
 
             return Ok(new Response(response, HttpStatusCode.OK));
         }
+
+        [HttpDelete]
+        [Route("{walletId}")]
+        public async Task<IActionResult> DeleteWalletAndOtpRequest(string walletId)
+        {
+            await _walletService.DeleteWallet(walletId);
+            return Ok(new Response(true, HttpStatusCode.OK));
```

Comment:
NOTE: I found it much better to use `StatusCode` over `Ok` because you can pass in any status code with the respective data

For future ref